ui/src/views/Roles/ViewRoles/Roles.jsx

```diff
@@ -30,14 +29,6 @@ import { VIEW_ROLES_PAGE_SIZE } from '../../../utils/constants';
   }),
 })
 export default class Roles extends PureComponent {
-  static propTypes = {
-    searchTerm: string,
-  };
-
-  static defaultProps = {
-    searchTerm: null,
-  };
```

Comment:
This function still reads the `searchTerm` prop so we should have it in `propTypes` and `defaultProps`.